src/main/resources/plugins.config

```diff
@@ -2,7 +2,7 @@
 # Author: Jan Eglinger
 # Version: 0.1.0
 
-Plugins>Blind Experiment, "Mask filenames", imagejan.plugins.Blind_Experiment("blind_files")
-Plugins>Blind Experiment, "Unmask filenames", imagejan.plugins.Blind_Experiment("unblind_files")
-Plugins>Blind Experiment, "Unmask results table", imagejan.plugins.Blind_Experiment("unblind_results")
+Plugins>Blind Experiment, "Mask filenames", sc.fiji.blind_experiment.Blind_Experiment("blind_files")
+Plugins>Blind Experiment, "Unmask filenames", sc.fiji.blind_experiment.Blind_Experiment("unblind_files")
+Plugins>Blind Experiment, "Unmask results table", sc.fiji.blind_experiment.Blind_Experiment("unblind_results")
```

Comment:
Now that all commands are implemented as SciJava `Command`s, this `plugins.config` is not needed any more. Let's remove it.